mypy_protobuf/main.py

```diff
@@ -269,8 +269,24 @@ def _write_comments(self, scl: SourceCodeLocation) -> bool:
         if sci_loc.trailing_comments is not None:
             lines.extend(self._break_text(sci_loc.trailing_comments))
 
+        lines = [
+            # Replace backslashes with double-backslashes
+            # to remove anything that would become an escape sequence in a Python string.
+            # Escape triple-quotes that would otherwise end the docstring early.
+            line
+                .replace("\\", "\\\\")
+                .replace('"""', '\\"\\"\\"')
```

Comment:
I actually suspect this will be insufficient. For example, I think if `"""""` is in a comment, this will end up as `\"\"\"""`.

```
>>> '"""""'.replace('"""', '\\"\\"\\"')
'\\"\\"\\"""'
```

I can think of two options:
1)
```
.replace('"', '\"')
```

2) Something like this
```
while '"""' in line:
  line = line.replace('"""', '\"\"\"')
```

Whatever you prefer.